waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -377,7 +377,7 @@ proc getMessagesV2ArbitraryQuery(
     endTime = none(Timestamp),
     maxPageSize = DefaultPageSize,
     ascendingOrder = true,
-): Future[ArchiveDriverResult[seq[ArchiveRow]]] {.async.} =
+): Future[ArchiveDriverResult[seq[ArchiveRow]]] {.async, deprecated.} =
```

Comment:
I'm not saying that is incorrect to label those procs as `deprecated` but how we should handle those? Do they still work/compile?

Reply:
I assumed it would only be a visual thing but I can't find any documentation so :shrug: They do compile/work though.